apps/chat/src/constants/errors.ts

```diff
@@ -9,7 +9,10 @@ export const errorsMessages = {
   400: 'Invalid request',
   contentFiltering:
     'The response was filtered due to the prompt triggering Azure OpenAI’s content management policy. Please modify your prompt and retry.',
-  unsupportedDataFormat: 'Unsupported data format',
+  unsupportedConversationsDataFormat:
+    'Import of prompts failed because of unsupported data format',
```

Comment:
Import of conversations I suppose
```suggestion
    'Import of conversations failed because of unsupported data format',
```